src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/ServiceDefaultsExtensions.cs

```diff
@@ -32,17 +44,19 @@ public static IHostApplicationBuilder AddServiceDefaults(this IHostApplicationBu
             http.AddServiceDiscovery();
         });
 
-        // Uncomment the following to restrict the allowed schemes for service discovery.
```

Comment:
Why is this removed? This block is here to make it very easy to discover how to limit service discovery to only allow HTTPS. This was agreed to as part of the security review for the related feature.